content/cloud/noops-sql-db.md

```diff
@@ -69,34 +67,41 @@ name = "c3"
 sqlite_databases = ["default"]
 ```
 
-The database will be created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+The above configuration will last the lifespan of the application.
 
-```bash
-$ spin cloud deploy
-Deploying app ..
-Creating database inspirational-pig (default) for app "quickstart"
-```
+### Preparing Fermyon Cloud Schema
 
-You can check to see your database by using the `spin cloud sqlite` command:
+Bootstrapping your Cloud SQLite database schema is done via the `spin cloud sqlite execute` subcommand. Your NoOps SQLite application on localhost will have already assigned a name for your database. All we need to do is obtain the name by typing the following command (in your local NoOps SQLite application's directory):
```

Comment:
The comment about `localhost` is very misleading.  None of this is on localhost - this is _specifically_ cloud stuff - and the name is not assigned by the application but by the cloud.

Reply:
So when a user runs `spin up --sqlite` and creates a new table (having not deployed to the cloud at all)
 the user's installed cloud plugin is the part that has done the work of creating the database table and subsequently the `spin cloud sqlite` subcommand is in the driver's seat?

Reply:
Thanks, this is resolved in the next push.